### PR TITLE
Fix the failing test cases

### PR DESCRIPTION
**Pull request by Bhashinee**
## Purpose
Fixes : https://github.com/ballerina-platform/ballerina-library/issues/6235

## Checklist
- [x] Linked to an issue
- [ ] Updated the specification
- [ ] Updated the changelog
- [ ] Added tests
- [ ] Checked native-image compatibility

### CODE REVIEW

**Review by NipunaRanasinghe — APPROVED** [review]

**Review by NipunaRanasinghe — COMMENTED** [review]
@Bhashinee can we update the CODEOWNERS file as well, as we still have choreo people as the owners? :)

**Comment by Bhashinee** [comment]
> @Bhashinee can we update the CODEOWNERS file as well, as we still have choreo people as the owners? :)

Saw the comment only after merging the PR. :) Sent a separate PR for this. Please review https://github.com/ballerina-platform/module-ballerinax-aws.sns/pull/59

**Pull request merged**
